controllers/topics.controller.js

```diff
@@ -10,3 +10,7 @@ module.exports.getTopics = (req, res, next) => {
       next(err);
     });
 };
+module.exports.getAPI = (req, res, next) => {
+  console.log(req.query, "in controller");
+  res.status(200).send(endpointsFile);
```

Comment:
you should send back all data from your controller on a meaningful key ie. `.send({endpoints: endpointsFile})`